packages/node/src/indexer/project.service.ts

```diff
@@ -253,6 +254,12 @@ export class ProjectService {
         'Specified project manifest chain id / genesis hash does not match database stored genesis hash, consider cleaning project schema using --force-clean',
       );
     }
+    if (this.project.network.bypassBlocks) {
```

Comment:
I think we should only upsert the blocks that has been bypassed, what if user changed this in manifest..

Previous bypassed can be changed